### PR TITLE
Add Kalray code drop refs in master branch, and a pointer to the last…

### PR DESCRIPTION
**Pull request by XavierRaynaud**
… one.

Also adapt build script and README accordingly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +56,15 @@ diff --git a/config.sub b/config.sub
...
 +++ b/config.sub
+@@ -2,7 +2,7 @@
+ # Configuration validation subroutine script.
+ #   Copyright 1992-2016 Free Software Foundation, Inc.
+ 
+-timestamp='2016-01-11'
++timestamp='2019-10-08'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
```
Comment (dkm) [inline README.md]: No need to re-add this...

File: README.md [hunk 2]
```diff
@@ -106,6 +117,15 @@ diff --git a/config.sub b/config.sub
...
 +++ b/config.sub
+@@ -2,7 +2,7 @@
+ # Configuration validation subroutine script.
+ #   Copyright 1992-2016 Free Software Foundation, Inc.
+ 
+-timestamp='2019-01-17'
++timestamp='2019-10-08'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
```
Comment (dkm) [inline README.md]: Same

File: README.md [hunk 3]
```diff
@@ -192,6 +212,15 @@ diff --git a/config.sub b/config.sub
...
 +++ b/config.sub
+@@ -2,7 +2,7 @@
+ # Configuration validation subroutine script.
+ #   Copyright 1992-2016 Free Software Foundation, Inc.
+ 
+-timestamp='2019-05-22'
++timestamp='2019-10-08'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
```
Comment (dkm) [inline README.md]: Same

File: README.md [hunk 4]
```diff
@@ -10,13 +10,15 @@ Kalray provides source codes of gdb/binutils, gcc and newlib that contain our po
 
 To build this toolchain, you have to clone github repository  https://github.com/kalray/build-scripts that contains build script and references of others needed repositories.
 These references correspond to official Kalray's deliveries.
+They are named "4.0.0-cd4.refs", "4.0.0-cd5.refs", etc...
```
Comment (dkm) [inline README.md:13]: 'etc' should not be followed by … ;)

**Review by dkm — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (XavierRaynaud) [reply README.md]: fixed. repushed.

**Pull request merged**